src/components/link/Link.stories.js

```diff
@@ -0,0 +1,63 @@
+import { html } from 'lit';
+import './link';
+import arrowRightIcon from '@carbon/icons/es/chevron--right/16';
+
+export default {
+  title: 'Components/Link',
+  component: 'kd-link',
+  argTypes: {
+    kind: {
+      control: { type: 'select' },
+      options: ['primary', 'secondary'],
+    },
+    target: {
+      control: { type: 'select' },
+      options: ['_self', '_blank', '_top', '_parent'],
+    },
+  },
+};
+
+const args = {
+  href: '',
+  target: '_self',
+  kind: 'primary',
+  disabled: false,
+};
+
+const linkIconArgs = { ...args, sizeOverride: 16 };
+
+// Inline link
+export const Inline = {
+  args: args,
+  render: (args) =>
+    html`
+      <kd-link
+        inline
+        href=${args.href}
+        target=${args.target}
+        kind=${args.kind}
+        ?disabled=${args.disabled}
+      >
+        Link1
```

Comment:
Add an arg for the `unnamed` slot so link text can be Contolled.

Reply:
Done. Referred with **button.ts** slots